app/components/modals/modal-base.js

```diff
@@ -7,12 +7,50 @@ import { isTesting } from 'open-event-frontend/utils/testing';
 
 export default UiModal.extend({
   tagName           : 'div',
+  classNames        : ['centered-modal'],
   classNameBindings : ['isFullScreen:fullscreen', 'isSmall:small', 'isLarge:large'],
 
   openObserver: observer('isOpen', function() {
     const $element = $(this.element);
     if (this.isOpen) {
-      $element.modal('show');
+      $element.modal({
+        centered       : false,
+        duration       : isTesting ? 0 : 200,
+        dimmerSettings : {
+          dimmerName : `${this.elementId}-modal-dimmer`,
+          variation  : 'inverted'
+        },
+        onHide: () => {
+          this.set('isOpen', false);
+          if (this.onHide) {
+            this.onHide();
+          }
+        },
+        onDeny: () => {
+          if (this.onDeny) {
+            this.onDeny();
+          }
+          return true;
+        },
+        onApprove: () => {
+          if (this.onApprove) {
+            this.onApprove();
+          }
+          return true;
+        },
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6392563)